service/models.py

```diff
@@ -175,13 +182,43 @@ def find(cls, by_params):
         logger.info("Processing lookup for id %s and condition %s ...", by_id, by_condition)
         return cls.query.get((by_id, by_condition))
 
-    # uncomment for sprint 2
-    # @classmethod
-    # def find_by_name(cls, name):
-    #     """Returns all Inventories with the given name
+    @classmethod
+    def find_by_name(cls, name):
+        """Returns all Inventories with the given name
+
+        Args:
+            name (string): the name of the Inventories you want to match
+        """
+        logger.info("Processing name query for %s ...", name)
+        return cls.query.filter(cls.name == name)
 
-    #     Args:
-    #         name (string): the name of the Inventories you want to match
-    #     """
-    #     logger.info("Processing name query for %s ...", name)
-    #     return cls.query.filter(cls.name == name)
+    @classmethod
+    def find_by_condition(cls, condition):
+        """Returns all Inventories with the given condition
+
+        Args:
+            condition (string): the condition of the Inventories you want to match
+        """
+        logger.info("Processing condition query for %s ...", condition)
+        return cls.query.filter(cls.condition == condition)
+
+    @classmethod
+    def find_by_quantity(cls, quantity):
```

Comment:
Is it a good idea to have quality filter like greater than or less than as well? Can we incorporate it here itself?

Reply:
I wonder how we would pass that in the url. Or actually we can remove this feature of `find _by_quantity` as logically no one would search by quantity standalone

Reply:
added operators less than, greater than, less than equal, greater than equal